presto-spi/src/main/java/io/prestosql/spi/connector/ConnectorSession.java

```diff
@@ -38,7 +39,15 @@ default String getUser()
 
     Optional<String> getTraceToken();
 
-    long getStartTime();
+    /**
+     * @deprecated use {@link #getStart()} instead
+     */
+    default long getStartTime()
```

Comment:
add `@Deprecated` too?

(should we have a static check?)

Reply:
What do you mean by a static check?

Reply:
Using deprecated in the Javadoc without the annotation.

Reply:
That makes sense. Is it even feasible to do that with checkstyle?